doc/architecture-quick-reference.rst

```diff
@@ -0,0 +1,468 @@
+Architecture Quick Reference
+============================
+
+This is a quick reference guide to IoTempower's architecture and key concepts.
+
+For detailed information, see:
+
+- `Architecture Overview <architecture.rst>`_ - Complete system architecture
+- `Device Architecture <device-architecture.rst>`_ - Device framework details
+- `Deployment Process <deployment-process.rst>`_ - Build and deploy workflow
+
+
+Key Concepts
+------------
+
+Systems, Nodes, and Devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+   IoT System (e.g., "my-home")
+   └── Nodes (e.g., "living-room", "bedroom")
```

Comment:
No, this is not a node, it's a room or collection of nodes (just a normal folder, without a node.conf or setup.cpp in it, but several node folders in it). In the bedroom can then be two bed reading lamps and other nodes. Maybe bedroom could have reading-lamp-left, reading-lamp-right, bed-leds, overhead-light, environment-reader. The nodes then have respective relay devicess (usually represented by an output device or using a sonoff basic that has a relay, button, and status led built in and available in examples) to turn on and off lamps, the bed-leds will have a local button and an rgb_strip_bus device connected, the environment-reader will have a dht22 sensor device for reading temperature and humidity.